tests/test_structures/test_bbox/utils.py

```diff
@@ -48,6 +48,9 @@ def is_inside(self, img_shape):
     def find_inside_points(self, points, is_aligned=False):
         pass
 
+    def create_masks(self, img_shape):
```

Comment:
typehint

Reply:
This line is a test code and other lines do not include the typehint.
Is it better to insert typehint in all these methods?